src/components/Select/Select.stories.mdx

```diff
@@ -0,0 +1,214 @@
+
+<Preview withToolbar>
+  <Story
+    name="Select"
+    args={{
+      id: "mySelect",
+      labelText: "What is your favorite color?",
+      showLabel: true,
+      helperText: "Choose wisely.",
+      errorText: "Waaaaaaaa!",
+      errored: false,
+      required: true,
+      disabled: false,
+      selectedOption: "Option Two",
+      attributes: {
+        onFocus: action("focus"),
+        onBlur: action("blur"),
+        onChange: action("onChange"),
+      },
+    }}
+  >
+    {args => (
+      <Select {...args}>
+        <option aria-selected={true}></option>
+        <option aria-selected={false}>Red</option>
+        <option aria-selected={false}>Green</option>
+        <option aria-selected={false}>Blue</option>
+        <option aria-selected={false}>Black</option>
+        <option aria-selected={false}>White</option>
+      </Select>
+    )}
+  </Story>
+</Preview>
+
+<ArgsTable story="Select" />
+
+## Labelling Variations
+
+A Select can be rendered with or without visible labels.
+
+<Preview withToolbar>
+  <Story
+    name="Labelling Variations"
+    args={{
+      labelText: "What is your favorite color?",
+      showLabel: false,
+    }}
+  >
+    {args => (
+      <Select {...args}>
+        <option aria-selected={true}></option>
+        <option aria-selected={false}>Red</option>
+        <option aria-selected={false}>Green</option>
+        <option aria-selected={false}>Blue</option>
+        <option aria-selected={false}>Black</option>
+        <option aria-selected={false}>White</option>
+      </Select>
+    )}
+  </Story>
+</Preview>
+
+<Canvas>
+  <Select labelText="What is your favorite color?" showLabel={true}>
+    <option aria-selected={true}></option>
+    <option aria-selected={false}>Red</option>
+    <option aria-selected={false}>Green</option>
+    <option aria-selected={false}>Blue</option>
+    <option aria-selected={false}>Black</option>
+    <option aria-selected={false}>White</option>
+  </Select>
+</Canvas>
+
+<Canvas>
+  <Select
+    labelText="What is your favorite color?"
+    showLabel={false}
+    helperText="Choose wisely!"
+  >
+    <option aria-selected={true}></option>
+    <option aria-selected={false}>Red</option>
+    <option aria-selected={false}>Green</option>
+    <option aria-selected={false}>Blue</option>
+    <option aria-selected={false}>Black</option>
+    <option aria-selected={false}>White</option>
+  </Select>
+</Canvas>
+
+<Canvas>
+  <Select
+    labelText="What is your favorite color?"
+    showLabel={true}
+    helperText="Choose wisely!"
+  >
+    <option aria-selected={true}></option>
+    <option aria-selected={false}>Red</option>
+    <option aria-selected={false}>Green</option>
+    <option aria-selected={false}>Blue</option>
+    <option aria-selected={false}>Black</option>
+    <option aria-selected={false}>White</option>
+  </Select>
+</Canvas>
+
```

Comment:
Similar idea here, it'd be good to say that `errorText` and `errored` were passed for the next examples.